app/README.md

```diff
@@ -36,17 +36,31 @@ The web build ensures that the UI as well as the data-fetching layer is fully ty
 
 ## Test
 
-The Phoenix app is statically analyzed for type safety via `typescript`, statically analyzed for best practices via [eslint](https://eslint.org/), and the formatting is enforced via the `prettier` code formatter. In addition to static type checking, the app also leverages [jest](https://jestjs.io/) as a unit testing framework. The following `pnpm` commands correspond to the above safeguards.
+The Phoenix app is statically analyzed for type safety via `typescript`, statically analyzed for best practices via [eslint](https://eslint.org/), and the formatting is enforced via the `prettier` code formatter. For unit testing, the app leverages [jest](https://jestjs.io/) as a unit testing framework. Lastly, for integration tests (e.g. end-to-end or e2e), [playwright](playwright.dev) is used. The following `pnpm` commands correspond to the above safeguards.
 
 ```shell
 pnpm run typecheck
 pnpm run lint
 pnpm run prettier:check
 pnpm test
+pnpm run test:e2e
 ```
 
 NB: [prettier](https://prettier.io/) is also enforced via [pre-commit](https://pre-commit.com/) hooks.
 
+### Integration Tests
+
+The integration tests run against the phoenix server built in "production" mode. Meaning you will have to have phoenix installed in your local python environment.
+
+```shell
+# from the root of the repository
+pip install -e .
```

Comment:
does this do anything? doesn't it get installed in teh testServer.mjs?

Reply:
No it doesn't. It assumes it's installed